source/a_releasenotes.rst

```diff
@@ -8,95 +8,87 @@ Release 3Di 8 March 2021
 The following has been released:
 
 - Local calculation of water depth maps
-- Extended support for calculations in the Modeller Interface
-- Extended API v3 with boundary conditions & bugfixing
-- Bugfixes in calculation core
-- Update land use map for damage estimations
+- Extended support for starting simulations using the Modeller Interface
+- Extended API v3 with boundary conditions & bug fixing
+- Bug fixes in the calculation core
+- Update land use map for the calculation of damage estimations
 
-Download the latest version of the Modeller Interface here. For QGIS users: upgrade the plugin using the plugin panel. The latest version is 1.16. 
+Download the latest version of the Modeller Interface here. For QGIS users: upgrade the plugin using the plugin panel. The latest version is 1.16. 
 
 Local calculation of water depth maps
 =====================================
 
-With the newest version of the Modeller Interface it is now possible to locally generate waterdepth maps for every time step of the calculation. 
-To generate these waterdepth maps from a quadtree calculation grid, 3Di applies a special algorithm to create visually appealing maps.
-With the plugin it is possible to calculate for both water level and water depth the interpolated and non-interpolated outputs of 3Di. 
+It is possible to generate water depth maps for every time step with the newest version of the Modeller Interface. To generate these water depth maps, 3Di applies a special algorithm that combines the water level results with the information of the DEM. This algorithm creates visually appealing maps. The maps show the water level and water depth results on high resolution, these can be based on the interpolated and on the non-interpolated water level results.
```

Comment:
Kunnen ze ook waterstandskaarten genereren vanuit MI?  Dan titel + tekst is niet consequent

Reply:
scherp. pas ik aan met een commit